sky/utils/schemas.py

```diff
@@ -629,9 +630,8 @@ def get_config_schema():
             'required': [],
             'additionalProperties': False,
             'properties': {
-                'security_group_name': {
-                    'type': 'string'
-                },
+                'security_group_name':
+                    (_PRORPERTY_NAME_OR_CLUSTER_NAME_TO_PROPERTY),
```

Comment:
Do we need this parenthesis?
```suggestion
                'security_group_name': _PRORPERTY_NAME_OR_CLUSTER_NAME_TO_PROPERTY,
```

Reply:
Unfortunately, black would fix to what you suggest, but then the linter would say the line is too long. I can add a commented exception if we want instead.

Reply:
Ahh, no worries then. I was thinking it might be within the length limit.